applications/vessel-history/src/features/event-filters/EventFilters.tsx

```diff
@@ -0,0 +1,97 @@
+import { useCallback, useState } from 'react'
```

Comment:
Thanks for generalizing this component and reusing it in both places Activity and the Map 💪